niworkflows/func/util.py

```diff
@@ -148,15 +170,14 @@ def init_bold_reference_wf(
         (inputnode, enhance_and_skullstrip_bold_wf, [
             ("bold_mask", "inputnode.pre_mask"),
         ]),
-        (inputnode, validate, [("bold_file", "in_file")]),
         (inputnode, gen_ref, [("sbref_file", "sbref_file")]),
         (inputnode, calc_dummy_scans, [("dummy_scans", "dummy_scans")]),
         (validate, gen_ref, [("out_file", "in_file")]),
         (gen_ref, enhance_and_skullstrip_bold_wf, [
             ("ref_image", "inputnode.in_file"),
         ]),
         (validate, outputnode, [
-            ("out_file", "bold_file"),
+            (("out_file", select_first), "bold_file"),
```

Comment:
I think we voted against using in-line functions within outputnode connections

Reply:
good catch - my bad!